frontend/src/components/common/Grid/components/NTag/style.ts

```diff
@@ -1,13 +1,12 @@
 import styled from "@emotion/styled";
-import { textClippingCSS } from "src/components/Collections/components/Grid/common/style";
 import { GRAY, PT_GRID_SIZE_PX } from "src/components/common/theme";
 import { CommonThemeProps, fontBodyXs, getColors, Tag as SDSTag } from "czifui";
 
 const gray100 = (props: CommonThemeProps) => getColors(props)?.gray[100];
 
 export const FieldValues = styled.div`
-  ${textClippingCSS}
```

Comment:
After the removal of all and any "My Collections" related components, the single remaining use of `textClippingCSS` on `FieldValues` was removed and replaced with its corresponding value; thereby allowing us to complete delete `src/components/Collections/components/Grid/common/style`.